src/coreclr/tools/aot/ILCompiler.TypeSystem.ReadyToRun.Tests/SignatureTests.cs

```diff
@@ -36,8 +36,8 @@ public SignatureTests(ITestOutputHelper output)
 
         private static string GetModOptMethodSignatureInfo(MethodSignature signature)
         {
-            if (!signature.HasEmbeddedSignatureData || signature.GetEmbeddedSignatureData() == null)
-                return "";
+            if (!signature.HasEmbeddedSignatureData)
```

Comment:
There is one more redundant check like this in src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs